app/src/main/res/layout/browser_overlay.xml

```diff
@@ -1,92 +1,89 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<org.mozilla.focus.browser.BrowserNavigationOverlayScrollView
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<org.mozilla.focus.browser.BrowserNavigationOverlayScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/overlayScrollView"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:fillViewport="true"
     android:focusable="false"
     android:scrollbars="none">
 
-    <LinearLayout
+    <android.support.v4.widget.NestedScrollView
```

Comment:
Is this for RecyclerView? I believe you can just set `app:layout_behavior="@string/appbar_scrolling_view_behavior"`

Reply:
Unfortunately, we get some weird scrolling behavior with that approach.

Reply:
Alas, then looks good!